src/tests/system/tests/test_access_control.py

```diff
@@ -0,0 +1,139 @@
+"""
+SSSD Authentication Test Cases
+
+:requirement: access control
+"""
+
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopologyGroup
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+@pytest.mark.parametrize("sssd_service_user", ("root", "sssd"))
+@pytest.mark.parametrize("method", ["su", "ssh"])
+@pytest.mark.importance("critical")
```

Comment:
@danlavu, what should be proper importance?

Reply:
'critical' or 'high', we do want these test to execute during tier1.

Reply:
I kept critical